django/core/management/templates.py

```diff
@@ -20,14 +20,15 @@
 
 class TemplateCommand(BaseCommand):
     """
-    Copy either a Django application layout template or a Django project
-    layout template into the specified directory.
+    Copy a Django template layout template into the specified directory.
+
+    Type of: app, project, (management) command
 
     :param style: A color style object (see django.core.management.color).
-    :param app_or_project: The string 'app' or 'project'.
-    :param name: The name of the application or project.
+    :param template_type: The string 'app', 'project', or 'command'.
```

Comment:
Perhaps alphabetizing the list would make the most sense, so "'app', 'command', or 'project'" ?